src/App.js

```diff
@@ -36,20 +36,20 @@ class App extends React.Component {
   searchYelp(term, location, sortBy) {
     Yelp.search(term, location, sortBy).then(businesses => {
       this.setState(
-        {
-          businesses: this.state.businesses
```

Comment:
Here state was being set to state so the results coming back were not replacing what was there.

Reply:
Hi Dom

Thanks so much for your help with this, it really is appreciated! Don't think I would have ever figured it out without you.

Chris

Reply:
You’re welcome. No problem at all. Should you have any other questions and want to learn anything more with code then just give me a shout. Happy to help as appreciate it can be a little overwhelming sometimes and usually a second pair of eyes can help 👍.

Cheers,
Dom